src/follow-me/edit.js

```diff
@@ -37,7 +39,18 @@ export default function Edit( { attributes, setAttributes } ) {
 					</PanelBody>
 				</InspectorControls>
 			) }
-			<FollowMe { ...attributes } id={ blockProps.id } />
+			{ isInheritMode ? (
+				<Card>
+					<CardBody>
+						{ createInterpolateElement(
+							__( 'This <strong>Follow Me</strong> block will adapt to the page it is on, showing the user profile if on a user archive, or the post author on a single post. It will be <strong>empty</strong> on non-author pages.', 'activitypub' ),
+							{ strong: <strong /> }
+						) }
+					</CardBody>
+				</Card>
+			) : (
```

Comment:
Could this show the actual block instead?

Reply:
There's nothing to show at this point because no user is selected. 

I thought people would be more likely to be confused by having one thing show in the Editor and something else on the frontend. I was also thinking of adding a fallback user selector and have that show, but again, the potential for confusion…

Reply:
I am not sure I follow. I fully agree, it should show the same thing in the editor as on the frontend. But I believe this can be achieved for cases when we talk about posts. The query loop block shows different posts live inside the editor, for example. The follow block could show for the right author for each post.

Reply:
Do you think it'd be reasonable to provide a preview when in a query loop? Good enough for a follow up, too, just wondering.

Reply:
Yeah I think making it show up in the loop would go a long way. I have no idea how to do that, should be interesting to learn. :)